### PR TITLE
Draft: Cold gas evolution data conversion

### PR DESCRIPTION
**Pull request by bwvdnbro**
Since I needed an easy way to add the cosmic HI/H2 abundance data to the new hypercube plots, I started implementing proper conversion scripts for these data, rather than reading the `.txt.` files directly from the plotting pipeline. This has the added advantage that we no longer need to obtain the cosmology from the snapshot, but can just use the `cosmology` that is passed on to the conversion script.

### CODE REVIEW

**Comment by MatthieuSchaller** [comment]
We should document the cosmology correction probably in the help string. That's something we added.

File: data/CosmicHIAbundance/conversion/convertPeroux2020.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+rhoHI = fitting_formula(zgrid, *raw[0])
+rhoHI_low = fitting_formula(zgrid, *raw[1])
+rhoHI_high = fitting_formula(zgrid, *raw[2])
+
+cosmology_correction = (
+    np.sqrt(cosmology.Om0 * (1.0 + zgrid) ** 3 + cosmology.Ode0) / cosmology.h
+) / (np.sqrt(0.3 * (1.0 + zgrid) ** 3 + 0.7) / 0.7)
+rhoHI *= 0.76 * cosmology_correction
+rhoHI_low *= 0.76 * cosmology_correction
+rhoHI_high *= 0.76 * cosmology_correction
```
Comment (MatthieuSchaller) [inline data/CosmicHIAbundance/conversion/convertPeroux2020.py:44]: Same with this Helium correction.

**Comment by bwvdnbro** [comment]
Done. Anything else you want changed?

**Review by MatthieuSchaller — APPROVED** [review]
That's now good for me. 

I'll let you merge it however if it is still relevant. I realise I should have acted earlier.

**Pull request merged**